enterprise/views.py

```diff
@@ -1067,18 +1066,16 @@ def get(self, request):
         if tpa_hint:
             # Add the tpa_hint to the redirect's 'next' query parameter
             # Redirect will be to the Enterprise Customer's TPA provider
-            tpa_next_param = {
-                'tpa_hint': tpa_hint,
-            }
-            query_dict['next'] = update_query_parameters(str(query_dict['next']), tpa_next_param)
+            query_dict['tpa_hint'] = tpa_hint
+            new_query = urlencode(query_dict, doseq=True)
         else:
             # If there's no linked IDP or multiple IDPs are linked and no tpa_hint provided in query_param
             # Add Enterprise Customer UUID and proxy_login to the redirect's query parameters
             # Redirect will be to the edX Logistration with Enterprise Proxy Login sidebar
             query_dict['enterprise_customer'] = [str(enterprise_customer.uuid)]
             query_dict['proxy_login'] = [True]
+            new_query = urlencode(query_dict, doseq=True)
```

Comment:
what is the point of moving and duplicating this code? I may be completely missing something, but neither part of the if/else returns something and it's not like you're defining a different "query_dict" variable.

Reply:
good catch it's a holdover from a prior round I forgot to address